src/bscPlugin/completions/CompletionsProcessor.ts

```diff
@@ -190,16 +190,16 @@ export class CompletionsProcessor {
             expression = file.getClosestExpression(this.event.position);
         }
 
-        if (!expression) {
-            return [];
-        }
-
         if (isFunctionExpression(expression)) {
             // if completion is the last character of the function, use the completions of the body of the function
             expression = expression.body;
         }
 
-        const tokenBefore = file.getTokenBefore(file.getClosestToken(expression.range.start));
+        if (!expression) {
```

Comment:
Moved this after the if statement above, because there was a small chance of expression being `null`, this catches that.